test/moveable-mixin-test.js

```diff
@@ -0,0 +1,56 @@
+/* global describe,beforeEach,afterEach,before,it:false */
```

Comment:
I think we could improve this. Probably eslint has some environment settings for mocha/chai.